HopsanGUI/LibraryHandler.h

```diff
@@ -54,6 +58,7 @@ class ComponentLibrary
 {
 public:
     QString id;
+    QString version;
```

Comment:
later we should try to reuse the library and component types from the generator library, but some linking changes are needed

Reply:
I agree. There are more specification classes that are duplicated in HopsanGUI, HopsanCore and HopsanGenerator.